products/BellHybrid/serial-number-parser/include/serial-number-parser/Common.hpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2017-2023, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#pragma once
+#include <string>
+
+namespace serial_number_parser
+{
+    constexpr auto grayColor     = "gray";
+    constexpr auto blackColor    = "black";
+    constexpr auto unknownColour = "unknown";
+
+    constexpr auto unknownSerialNumber = "0000000000000";
+
+    constexpr auto firstVersion   = 1;
+    constexpr auto secondVersion  = 2;
+    constexpr auto unknownVersion = 0;
+
+    /* Serial number and device version */
```

Comment:
This comment isn't needed here I think ;)

Reply:
done